### PR TITLE
Users can select tags on search page, project tags update

### PR DESCRIPTION
**Pull request by khanma92**

### CODE REVIEW

File: server/src/models/index.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const Project = ProjectModel(sequelize, Sequelize)
 // Sets up the associations between Tag and User (a many to many relationship).
 const Users_Tag = sequelize.define('Users_Tag', {}, { timestamps: false} )
 const Users_Project = sequelize.define('Users_Project', {}, {timestamps: false} )
-const Project_Tag = sequelize.define('Projects_tag', {}, {timestamps: false})
+const Project_Tag = sequelize.define('Projects_tags', {}, {timestamps: false})
```
Comment (VeerpalBrar) [inline server/src/models/index.js]: why did you change the database name?

Comment (khanma92) [reply server/src/models/index.js]: I don't know why but it only works (for me) when the database name is 'Projects_tags'. I've tried deleting my database as well, but I'm not sure why I should still have to change the name. I'll try to figure it out

Comment (khanma92) [reply server/src/models/index.js]: > why did you change the database name?
I don't know why but it only works (for me) when the database name is 'Projects_tags'. I've tried deleting my database as well, but I'm not sure why I should still have to change the name. I'll try to figure it out

Comment (VeerpalBrar) [reply server/src/models/index.js]: I just checked my local db, and `Project_tags` has no entries. I think since we are never saving project tags in the db on master, the   `Project_tags` model is not referenced and so we don't get the error. So this may not be unique to you. I'll test this a bit more tomorrow and get back to you.

Comment (VeerpalBrar) [reply server/src/models/index.js]: Never mind, I tested it this morning, and using `Project_tag` works fine for me now. Can you leave this as `Project_tag` for now and we can investigate further later? That we can still merge this PR without affecting everyone's db.

File: client/src/components/PostProjects.vue [hunk 2]
```diff
@@ -80,16 +80,25 @@
...
                 var form = new FormData();
                 form.append('picture', this.file);
                 form.append('userid', 1);
                 form.append('desc', this.description);
                 form.append('project_name', this.name);
+                form.append('tags', JSON.stringify(this.value));
                 axios.post('/connect/post-projects', form , {headers: {'Content-Type': 'multipart/form-data'}})
                 .then(response => {
-                        this.projectname= response.data.project_name;
+                        this.project_name= response.data.project_name;
```
[the remark this answers is not in the document]
Comment (khanma92) [reply client/src/components/PostProjects.vue]: > shouldn't this be this.name instead of this.project_name?

I dont think it matters, I just tried it with this.name and the database is still getting updated.

[the remark this answers is not in the document]
Comment (ishaaa-ai) [reply client/src/components/PostProjects.vue]: oh alright! then i guess it should be fine.
